apache/superset
# chore: Select component refactoring - ColorSchemeControl - Iteration 5

### SUMMARY
It replaces the react-select Select component with the Antdesign one in the ColorSchemeControl component.

### BEFORE
https://user-images.githubusercontent.com/60598000/124637377-f6f3f100-de89-11eb-835f-7be93edb5d68.mp4

### AFTER
https://user-images.githubusercontent.com/60598000/124637400-0115ef80-de8a-11eb-93ec-3deec4e1f265.mp4

### TESTING INSTRUCTIONS
1. Open a chart, for instance, the _Cross Channel Relationship_ example chart
2. Click on the "Customize" tab
3. Observe the COLOR SCHEME Select
4. Select a different color scheme and observe the behavior

### ADDITIONAL INFORMATION
